docs-site/docs/extend/graphql.md

```diff
@@ -0,0 +1,215 @@
+          "id": "gid://irida/Sample/2"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 3",
+          "id": "gid://irida/Sample/3"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 4",
+          "id": "gid://irida/Sample/4"
+        },
+        {
+          "name": "Bacillus anthracis/Outbreak 2022 Sample 5",
+          "id": "gid://irida/Sample/5"
+        }
+      ]
+    }
+  }
+}
+```
+
+</td>
+</tr>
+
+<tr>
+<td> Get a specific samples details </td>
+<td>
+
+```graphql
+query {
+  node(id: "gid://irida/Sample/1"){
+    ... on Sample{
+      id,
+      name
+      description
+      metadata
+      project{
+        id,
+        fullPath
+      }
+    }
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "data": {
+    "node": {
+      "id": "gid://irida/Sample/1",
+      "name": "Bacillus anthracis/Outbreak 2022 Sample 1",
+      "description": "This is a description for sample Bacillus anthracis/Outbreak 2022 Sample 1.",
+      "project": {
+        "id": "gid://irida/Project/1",
+        "fullPath": "bacillus/bacillus-anthracis/outbreak-2022"
+      },
+      "metadata": {
+        "age": 40,
+        "food": "Cheeseburger",
+        "onset": "2022-06-21",
+        "WGS_id": 6862301436,
+        "gender": "Female",
+        "country": "Gabon",
+        "patient_age": 8,
+        "patient_sex": "Male",
+        "earliest_date": "2022-10-03",
+        "NCBI_ACCESSION": "NM_7807606.5"
+      }
+    }
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+## Authentication with Personal Access Tokens
+
+To use the GraphQL API with [any of the GraphQL Clients](https://graphql.org/code/#graphql-clients), you will need to generate a Personal Access Token.
+
+### Navigate to the Access Tokens screen
+
+When you are signed in to IRIDA Next, click `Edit profile`,
+
+![Edit Profile Button](edit_profile.png)
+
+and then select `Access Tokens` from the sidebar.
+
+![Access Tokens Button](access_tokens.png)
+
+### Token Scopes
+
+There are 2 scopes for your access token, `api` and `read_api`
+
+If your GraphQL queries will only be reading data, and not making any changes, select `read_api`.
+
+If your GraphQL queries will be modifying data or uploading new data, select `api`.
+
+### Generate a Token
+
+When `Create personal access token` is clicked, a secret token will be generated for you. This will be used to authenticate your GraphQL API requests.
+
+It is important that you do not share this token with anyone else as it is directly tied to your account.
+
+### Using Token for Sessionless Authentication
+
+Tokens generated by IRIDA Next are `Basic Authentication` tokens. They are tied to your email.
+
+This uses [Basic HTTP Authentication](https://datatracker.ietf.org/doc/html/rfc7617), in the form of $USEREMAIL:$ACCESSTOKEN that is Base64 encoded.
+
+e.g. `admin@email.com:yK1euURqVRtQ1D-3uKsW`` becomes `YWRtaW5AZW1haWwuY29tOnlLMWV1VVJxVlJ0UTFELTN1S3NX``
```

Comment:
**nitpick (blocking):** You have extra backticks here which is causing formatting errors when rendering the documentation.

Reply:
Fixed.